SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/OntologyMapper.Mapped.csproj

```diff
@@ -8,8 +8,8 @@
 		<Version>$(VersionPrefix)</Version>
 		<Authors>Microsoft</Authors>
 		<Description>Support for converting from one Mapped Ontology to a different DTDL Based Ontology</Description>
-		<AssemblyVersion>0.9.1</AssemblyVersion>
-		<PackageVersion>0.9.1-preview</PackageVersion>
+		<AssemblyVersion>0.9.19</AssemblyVersion>
```

Comment:
Why such a large jump?

Reply:
Have been testing locally. That's how many tries it took me to get it right